src/pynxtools/dataconverter/readers/multi/reader.py

```diff
@@ -343,11 +387,21 @@ def get_processing_order(path: str) -> Tuple[int, Union[str, int]]:
         self.post_process()
 
         if self.config_dict:
-            template.update(
-                fill_from_config(
-                    self.config_dict, self.get_entry_names(), self.callbacks
+            if "suppress_warning" in kwargs:
+                template.update(
+                    fill_from_config(
+                        self.config_dict,
+                        self.get_entry_names(),
+                        self.callbacks,
+                        kwargs["suppress_warning"],
+                    )
+                )
+            else:
+                template.update(
+                    fill_from_config(
+                        self.config_dict, self.get_entry_names(), self.callbacks
+                    )
```

Comment:
Can be compacted a bit

```suggestion
            template.update(
                fill_from_config(
                    self.config_dict,
                    self.get_entry_names(),
                    self.callbacks,
                    **({'suppress_warning': suppress_warning} if "suppress_warning" in kwargs else {})
                )
            )
```

Reply:
Went with `**{k: v for k, v in kwargs.items() if k == "suppress_warning"}`, I think that's even more concise.